Multisig/Logic/Models/Transaction/Transaction.swift

```diff
@@ -41,6 +41,11 @@ struct Transaction: Decodable, Hashable {
     var signatures: DataString?
     var transfers: [TransactionTransfer]?
     var txType: TransactionType?
+
```

Comment:
I am confused. Why are there two "Transaction" structs?

Reply:
which two structs?

Reply:
we have `struct Transaction` in `Multisig/Logic/Models/Transaction.swift ` and in `Multisig/Logic/Models/Transaction/Transaction.swift`

Reply:
I don't understand, I can't find the second one

Reply:
this is from the merge after refactoring, the file `Multisig/Logic/Models/Transaction.swift` needs to be deleted

Reply:
I can see only one on my side
![Screen Shot 2020-06-18 at 1 24 43 PM](https://user-images.githubusercontent.com/58591867/85015046-9b17d680-b167-11ea-8d89-cda74907bfd1.png)

Reply:
check the file system, not the Xcode